Constitution.md

```diff
@@ -97,32 +101,102 @@ The Terms of Office for all Executive Positions at CSSA will last for at most on
 
 ## Article 2b: Executive Officers and Duties
 
-### President(s)
+
+### Co-Presidents
+
+The Co-presidents is a transitional role approved by the 2022-2023 council for the 2023-2024 CSSA general elections and elected as a single voting member on council.
 
 1. Act as a signing authority for CSSA.
 2. Be the contact person for UMSU.
-3. Attend Student Executive Assembly (SEA) Meetings on behalf of CSSA.
-4. Be responsible for the overall operations of the group.
-5. Deal with issues concerning the club and its individual Members.
-6. Serve as signing authority on any group accounts.
-7. Be a voting member of CSSA for management decisions.
-8. Call special meetings.
-9. Attend all meetings.
-10. Carry out provisions of the constitution.
-11. Perform any other duties incidental to the office of the President or directed by Council.
+3. Maintain frequent and punctual communication with the Faculty of Science, Department of Computer Science, UMSU, Members of Council, Members of the Association, and all other relevant parties.
+4. Attend Student Executive Assembly (SEA) Meetings on behalf of CSSA.
+5. Be responsible for the overall operations of the group.
+6. Deal with issues concerning the club and its individual Members.
+7. Serve as signing authority on any group accounts.
+8. Be a voting member of CSSA for management decisions.
+9. Call special meetings.
+10. Attend all meetings.
+11. Carry out provisions of the constitution.
+12. Ensure the group funds remain in budget with local and corporate sponsors along with the Director of Finances and Internal Affairs.
+13. Ensure that the CSSA lounge is a safe and inclusive space.
+14. Enforce that the transition report is done by April 30 of the council’s term.
+15. Perform any other duties incidental to the office of the President or directed by Council.
+
+### Director of Finances and Internal Affairs
+
+1. Act as a signing authority for CSSA.
+2. Providing a brief report on CSSA finances and expenses at each meeting.
+3. Be responsible for all funds generated by Council events and for overseeing all funds spent by Council.
+4. Make available all audits of CSSA to any Student members upon request with more than two weeks of notice.
+5. Ensure the Council is kept updated on ongoing council initiatives.
+6. Perform a year-end audit at the end of the CSSA Fiscal Year.
+7. Reserve the meeting room for CSSA meetings, General meetings and the CS Executive Assembly.
+8. Prepare an agenda and council package for Regular Meetings and distribute them to the Council at least two days prior to the meeting.
+9. Receive resignations of Council and inform Executives and the Computer Science student body of any such resignations.
+10. Ensure that the CSSA lounge is a safe and inclusive space.
+11. Lead the Financial Accounability Committee with up to 3 committee members.
```

Comment:
Why do we have an upper limit on the number of committee members. Why not leave it open and the Finance and Internal Affairs exec can decide that?